### PR TITLE
fix: UUID auto generate; Error handling & e2e tests

### PR DESCRIPTION
**Pull request by filipecabaco**
## What kind of change does this PR introduce?

* UUID auto generation on migration
* Better error handling
* Fix e2e tests

### CODE REVIEW

File: test/e2e/tests.ts [hunk 1]
```diff
@@ -275,11 +291,16 @@ describe("authorization check", () => {
...
   const table = "broadcast_changes";
-  const id = 1;
+  const id = crypto.randomUUID();
 
   it("authenticated user receives insert broadcast change from a specific topic based on id", async () => {
     let supabase = await createClient(url, token, { realtime });
-    let accessToken = await signInUser(supabase, "test1@test.com", "test_test");
+    let accessToken = await signInUser(
+      supabase,
+      "filipe@supabase.io",
```
Comment (J0) [inline test/e2e/tests.ts:300]: /nit Not sure what the realtime stance is but in Auth we generally exclude references to @supabsae.io

Can consider changing the test email address

**Review by J0 — APPROVED** [review]
Looks like we move the auto generation logic from Ecto to Postgres, add additional error handling and a few e2e tests

Nice work.

File: test/e2e/tests.ts (hunk 1 above)
Comment (filipecabaco) [reply test/e2e/tests.ts:300]: we need it for Supabase staging projects 😓

**Pull request merged**